packages/backend/src/tenants/service.test.ts

```diff
@@ -42,9 +43,13 @@ describe('Tenant Service', (): void => {
   let config: IAppConfig
   let apolloClient: ApolloClient<NormalizedCacheObject>
   let knex: Knex
+  const dbSchema = 'tenant_service_test_schema'
```

Comment:
This allows us to have the tenant service tests run cleanly without dropping the tenants table other tests are using

Reply:
In my PR, I have updated the truncateTables to not delete the tenant. 
https://github.com/interledger/rafiki/pull/3152

Reply:
Same [here](https://github.com/interledger/rafiki/pull/3206/files#diff-0bf515169bf402f4617999baf3720eaffe164a7346568fbd615c43a80b96f2ea). I also added a way to configure db schema, because for the tenant service tests, we _do_ want to truncate the tenants table. But then of course, we break all of the other tests :) so I needed to isolate it somehow

Reply:
I like that. I will merge yours into mine @mkurapov